service.onmute/settings.py

```diff
@@ -0,0 +1,24 @@
+#   Copyright (C) 2017 Lunatixz
+#
+#
+# This file is part of OnMute
+#
+# OnMute is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# OnMute is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with OnMute.  If not, see <http://www.gnu.org/licenses/>.
+
+import xbmcaddon
```

Comment:
Yes :) it's a "habbit"/ code style to declare default values. You'll also note I prefer the style " if var == True:" over the short, "If var:" I really hope my minor style preferences aren't a issue. :)